admin/zabbix/Makefile

```diff
@@ -222,16 +230,30 @@ define Package/zabbix-server/install
 	$(call Package/zabbix/install/etc,$(1),server)
 endef
 
+define Package/zabbix-server-frontend/install
+	$(INSTALL_DIR) $(1)/www/zabbix
+	$(CP) $(PKG_BUILD_DIR)/frontends/php/* $(1)/www/zabbix
+endef
+
 define Package/zabbix-proxy/install
 	$(call Package/zabbix/install/sbin,$(1),proxy)
 	$(call Package/zabbix/install/etc,$(1),proxy)
 endef
 
+define Package/zabbix-server-frontend/postinst
+#!/bin/sh
+[ -n "$${IPKG_INSTROOT}" ] || {
+	ln -sf /www/zabbix/ /www/zabbix/
```

Comment:
what is this for?

Reply:
Yes, but you're using the same path twice.  are you _sure_ this is correct.  It's not from the listed reference either.

Reply:
This solution worked before. I corrected it to stop mislead people. Now it should be okay. I have tested it and it works perfect after adding php interpreter to uhttpd.